whence: add page (#4021)
tldr-pages/tldr: pages/osx/whence.md

# whence

> A zsh builtin to indicate how a given command would be interpreted.

> - Interpret {{command}}, with expansion if defined as an `alias` (similar to `command -v` builtin):

Reviewer: Perhaps this will read better?
```suggestion
- Interpret {{command}}, with expansion if defined as an `alias` (similar to the `command -v` builtin):
```

`whence {{command}}`

> - Display type of {{command}}, with location if defined as a function, or binary (equivalent to `type` and `command -V` builtins):

Reviewer: Perhaps this will read better?
```suggestion
- Display type of {{command}}, with location if defined as a function, or binary (equivalent to the `type` and `command -V` builtins):
```

`whence -v {{command}}`

> - Same as above, except display content of shell functions instead of location (equivalent to `which` builtin):

Reviewer: The others are builtins, but I don't think `which` is (try `which which` and see for yourself). Perhaps we could update this - what about this?
```suggestion
- Same as above, except display content of shell functions instead of location (equivalent to the `which` command):
```

Author: `which` is a shell builtin, as well as `/usr/bin/which`. If you're on macOS like me, you can do `builtin whence -va which` and see for yourself @sbrl…

Author: @sbrl I've clarified this in another entry; please see latest changes…

`whence -c {{command}}`

> - Same as above, but show all occurrences on command path (equivalent to `where` builtin):

Reviewer: Perhaps this will read better?
```suggestion
- Same as above, but show all occurrences on command path (equivalent to the `where` builtin):
```
